code/controllers/configuration.dm

```diff
@@ -81,6 +81,7 @@
 	var/forumurl = "http://baystation12.net/forums/"
 
 	var/media_base_url = "http://nanotrasen.se/media" // http://ss13.nexisonline.net/media
+	var/overflow_server_url = "byond://nanotrasen.se:6666"

```

Comment:
Nothing should go wrong. If it's not set, it won't redirect people. Having it set to Paradise means, in a worst case scenario, someone else accidentally sets their server up to redirect to us. That's bad. Don't do that.